_docs/tasks/configuring-mixer.md

```diff
@@ -0,0 +1,167 @@
+---
```

Comment:
So reading through this, I don't know if this really qualifies as a task. A task is something fairly targeted in nature. For example: "turning on rate limits". It's not intended for general descriptions, that's what the conceptual docs are for.

Now as a concept doc, this doesn't contain enough information to be useful. For example, the doc mentions selectors but doesn't actual say how they work, how to construct expressions, etc.

So I think we either want to cut this down into specific tasks like "Registering a new Adapter", or we need to beef it up and really be a guide-level document to move into the Concepts section.